ui/src/app/pages/workspace/research-purpose.tsx

```diff
@@ -100,7 +100,7 @@ export const ResearchPurpose = fp.flow(
   return (
     <FadeBox>
       <div style={styles.mainHeader}>
-        Primary purpose of project
+        Research Use Statement Questions
```

Comment:
The edit button was floating near the name of the first question. It seems more intuitive to have it next to an overarching header

Reply:
This component is also largely similar to research-purpose-section (admin page). By moving the edit button away from the title of the first section, this becomes much easier to share.